specification/flag-evaluation/flag-evaluation.json

```diff
@@ -62,26 +62,26 @@
     {
         "id": "1.13",
         "clean id": "1_13",
-        "content": "The `evaluation details` structure's `variant` field SHOULD contain a identifier logically corresponding to the return flag value.",
-        "RFC 2119 keyword": "SHOULD"
+        "content": "In cases of normal execution, the `evaluation details` structure's `variant` field MUST contain the value of the `variant` field in the `flag resolution` structure returned by the configured `provider`.",
+        "RFC 2119 keyword": "MUST"
     },
     {
         "id": "1.14",
         "clean id": "1_14",
-        "content": "The `evaluation details` structure's `reason` field SHOULD indicate the semantic reason for the returned flag value. Possible values vary by provider, but may include `\"TARGETING_MATCH\"`, `\"SPLIT\"`, `\"DISABLED\"`, `\"DEFAULT\"`, `\"UNKNOWN\"` and `\"ERROR\"`.",
-        "RFC 2119 keyword": "SHOULD"
+        "content": "In cases of normal execution, the `evaluation details` structure's `reason` field MUST contain the value of the `reason` field in the `flag resolution` structure returned by the configured `provider`.",
+        "RFC 2119 keyword": "MUST"
     },
     {
         "id": "1.15",
         "clean id": "1_15",
-        "content": "In cases of normal execution, the `evaluation details` structure's `error code` field MUST not be set, or otherwise must contain a null or falsy value.",
+        "content": "In cases of normal execution, the `evaluation details` structure's `error code` field MUST contain the value of the `error code` field in the `flag resolution` structure returned by the configured `provider`.",
```

Comment:
Errr... in normal execution, there shouldn't be an error!

Reply:
See discussion here: https://github.com/open-feature/spec/pull/41#discussion_r860195055, duplicated.